resources/kcp/charts/kyma-environment-broker/templates/subscription-cleanup-job.yaml

```diff
@@ -30,5 +30,5 @@ spec:
             - name: gardener-kubeconfig
               secret:
                 secretName: {{ .Values.gardener.secretName }}
-  schedule: '0 2 * * *'
+  schedule: '0 2,14 * * *'
```

Comment:
It implies to run subscription-cleanup-job every 12 hours which is IMHO too rare. You should restore previous schedule for this job after problem is fixed.

Reply:
just a question: I'm not sure how statistics looks like, how is it possible to have all subscriptions dirty, but why not run such job every hour?